src/foundry/client/data/documents/token.d.ts

```diff
@@ -214,37 +225,22 @@ declare global {
      * Get an Array of attribute choices which could be tracked for Actors in the Combat Tracker
      * @param _path - (default: `[]`)
      */
-    static getTrackedAttributes(
-      data?: InstanceType<ConfiguredDocumentClass<typeof Actor>>["data"]["data"],
-      _path?: string[]
-    ): TrackedAttributes;
+    static getTrackedAttributes(data?: ActorData, _path?: string[]): TrackedAttributes;
 
     /**
      * Inspect the Actor data model and identify the set of attributes which could be used for a Token Bar
+     * @param attributes    - The tracked attributes which can be chosen from
+     * @returns A nested object of attribute choices to display
      */
-    static getTrackedAttributeChoices(attributes?: TrackedAttributes): Record<string, string[]>;
+    static getTrackedAttributeChoices(attributes: TrackedAttributes): Record<string, string[]>;
   }
+}
 
-  /**
-   * An extended Document definition used specifically
-   * This ensures that the PrototypeTokenData schema is used instead of the standard TokenData.
-   * This exists specifically for prototype Token configuration in the TokenConfig app and should not be used otherwise.
-   * @internal
-   * @remarks The types for this are incorrect, in particular everything related to the data should use
-   * {@link foundry.data.PrototypeTokenData} instead of {@link foundry.data.TokenData}. However, with the current approach
-   * to type documents, we don't see a way to do this. This class is just added for reference. As the documentation above
-   * says, don't use it (there really shouldn't be a need to do so).
-   */
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  class PrototypeTokenDocument extends TokenDocument {
```

Comment:
I completely removed the deprecated PrototypeTokenDocument section because it was only throwing an error. Do we need it to be there, regardless?

Reply:
It only needs to be there for as long as source has it.

Reply:
Ah, also, I read the code wrong. I thought it was just throwing an error. I must have been reading v11 code in that moment 😬

Regardless, I've added it back